internal/rpc/jsonrpc/methods.go

```diff
@@ -3005,12 +3000,7 @@ func (s *Server) listReceivedByAddress(ctx context.Context, icmd interface{}) (i
 			for _, cred := range tx.Credits {
 				pkVersion := tx.MsgTx.TxOut[cred.Index].Version
 				pkScript := tx.MsgTx.TxOut[cred.Index].PkScript
-				_, addrs, _, err := txscript.ExtractPkScriptAddrs(pkVersion,
-					pkScript, w.ChainParams(), true) // Yes treasury
-				if err != nil {
-					// Non standard script, skip.
-					continue
```

Comment:
There seemed to be an incorrect assumption in many plcaes that non-nil error meant non-standard script, but that was not correct.  Depending on the use, I either removed any checks, or added a check on either `class` or `addrs` depending on the intent I could discern from the context.